indicators/validators.py

```diff
@@ -0,0 +1,65 @@
+import string
```

Comment:
I guess this module is redundant (see my comments to the `CountryIndicatorForm` in `indicators/forms.py`)